test-app-conf/Migrations/20221208175645_DropProxyTable.php

```diff
@@ -0,0 +1,20 @@
+<?php
+declare(strict_types=1);
+
+use Eggheads\CakephpCommon\Phinx\Migration\AbstractMigration;
+
+class DropProxyTable extends AbstractMigration
+{
+    /**
+     * Change Method.
+     *
+     * More information on this method is available here:
+     * https://book.cakephp.org/phinx/0/en/migrations.html#the-change-method
+     *
+     * @return void
+     */
+    public function change(): void
+    {
+        $this->table('proxy_config')->drop()->save();
+    }
```

Comment:
Лишняя документация, также использование метода change тут не совсем уместно, drop является irreversible операцией в phinx - лучше написать полноценную логику в up/down.
```suggestion
    /**
     * @inheritDoc
     */
    public function up(): void
    {
        $this->table('proxy_config')->drop()->save();
    }

    /**
     * @inheritDoc
     */
    public function down(): void
    {
        // ...
    }
```